client/backend.js

```diff
@@ -212,5 +212,17 @@ module.exports = {
             success: cb,
             error: xhrError
         });
+    },
+    
+    setTypeForOperation: function(operationId, operationTypeId, cb) {
```

Comment:
I'll let it like that as I don't knoew the promise model. If you point to the API, I can correct now.